### PR TITLE
[NONE] cutting release 0.76.0

### PR DESCRIPTION
**Pull request by nicolastakashi**
## Description

_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue._

## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Verification

Please check the [Prometheus-Operator testing guidelines](../TESTING.md) for recommendations about automated tests.

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+## 0.76.0 / 2025-08-08
+
+[FEATURE] OAuth2 TLSConfig and ProxyConfig settings. #6735
+[ENHANCEMENT] Use service endpoint role for alerting. #6784
+[FEATURE] Support configuring OAuth2 in ScrapeConfig. #6814
+[FEATURE] Add scale subresource to Alertmanger resource. #6728
+[ENHANCEMENT] set the namespace if empty for alertmanagers to the current namespace in Prometheus CR. #6338
+[ENHANCEMENT] support TLSConfig TLSVersion setting. #6736
```
Comment (simonpasquier) [inline CHANGELOG.md:8]:
```suggestion
[ENHANCEMENT] Add support to configure the TLS version for Prometheus, PrometheusAgent and Alertmanager CRDs. #6736
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+[ENHANCEMENT] optimize `TLSConfig` to `addSafeTLSToYaml`. #6726
```
Comment (simonpasquier) [inline CHANGELOG.md:9]: no need to mention IMHO
```suggestion
[ENHANCEMENT] optimize `TLSConfig` to `addSafeTLSToYaml`. #6726
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+[ENHANCEMENT] Add ServiceDiscoveryRole configuration for use of Endpoints vs EndpointSlice. #6672
```
Comment (simonpasquier) [inline CHANGELOG.md:10]: It's more of a feature for me.
```suggestion
[FEATURE] Add `serviceDiscoveryRole` field to the Prometheus and PrometheusAgent CRDs to select between Endpoints (default) and EndpointSlice for discovering scrape and alerting targets. #6672
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,19 @@
...
+[ENHANCEMENT] Add secret label selector to filter secrets to watch. #6731
```
Comment (simonpasquier) [inline CHANGELOG.md:11]:
```suggestion
[ENHANCEMENT] Add `secret-label-selector` argument to the operator to filter the Secrets being watched. #6731
```

**Review by simonpasquier — COMMENTED** [review]
This release is pretty packed :partying_face:

**Comment by simonpasquier** [comment]
It might be worth mentioning https://github.com/prometheus-operator/prometheus-operator/pull/6678 too because it adds more validations to the Kubernetes SD of ScrapeConfig CRD (not necessary to mention them exhaustively).

**Comment by ArthurSens** [comment]
@nicolastakashi and I talked and he will create a new branch to fix the work here.

**Pull request closed without merging**